mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/JPAMailboxManagerTest.java

```diff
@@ -49,9 +41,8 @@ protected MailboxManager provideMailboxManager() {
         return openJPAMailboxManager.get();
     }
 
-    @Override
-    @After
-    public void tearDown() throws MailboxException {
+    @AfterEach

```

Comment:
Not sure it plays well with the same annotation in MailboxManagerTest. Previous strategy is probably more predictable, don't you think ?

Reply:
> Not sure it plays well with the same annotation in MailboxManagerTest. Previous strategy is probably more predictable, don't you think ? 

I checked, it's working like a charm.

Reply:
parent run first ?

Reply:
Yes